packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -152,12 +147,12 @@ export class CommandBarBase extends BaseComponent<ICommandBarProps, {}> implemen
     if (item.iconOnly && itemText !== undefined) {
       return (
         <TooltipHost content={itemText} {...item.tooltipHostProps}>
-          <CommandButtonType {...commandButtonProps as IButtonProps} />
+          <ItemCommandButtonType {...commandButtonProps as IButtonProps} defaultRender={CommandButtonType} />
```

Comment:
Why do we need the default render? Its just the CommandBarButton.

Reply:
After discussing with @ThomasMichon we decided it would be better if the consumer didn't need to worry about what the default component is and just be able to render it inside of what ever additional components we add.

There is a similar pattern used by the IRenderFunction interface:
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/utilities/src/IRenderFunction.ts
 which is used for rendering subMenuProps for the onRenderSubMenu here:
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

Reply:
Yeah, i'm fine with the idea of passing the default component through, though i'm curious if we should say "defaultRender" vs "defaultComponent" or something.